deeplake/core/tensor.py

```diff
@@ -1687,26 +1687,15 @@ def load_vdb_index(self, id: str, path: Optional[str] = None):
         ts = getattr(ds, self.meta.name)
         from indra import api  # type: ignore
```

Comment:
This and also lines to get `commit_id` and `b` also should be removed.

Reply:
Done.